apps/etterlatte-vedtaksvurdering/src/main/kotlin/no/nav/etterlatte/vedtaksvurdering/VedtakKlageService.kt

```diff
@@ -97,26 +99,75 @@ class VedtakKlageService(
         sjekkAttestantHarAnnenIdentEnnDenSomFattet(eksisterendeVedtak, brukerTokenInfo)
         verifiserGyldigVedtakStatus(eksisterendeVedtak.status, listOf(VedtakStatus.FATTET_VEDTAK))
 
-        return repository.attesterVedtak(
-            klage.id,
-            Attestasjon(
-                attestant = brukerTokenInfo.ident(),
-                attesterendeEnhet = klage.sak.enhet,
-                tidspunkt = Tidspunkt.now(),
-            ),
-        ).id
+        val attestertVedtak =
+            repository.attesterVedtak(
+                klage.id,
+                Attestasjon(brukerTokenInfo.ident(), klage.sak.enhet, Tidspunkt.now()),
+            )
+        try {
+            rapidService.sendToRapid(vedtakOgRapidAttestert(attestertVedtak))
+        } catch (e: Exception) {
+            logger.error(
+                "Kan ikke sende attestert vedtak på kafka for behandling id: ${attestertVedtak.behandlingId}, " +
+                    "vedtak: ${attestertVedtak.id}, " +
+                    "saknr: ${attestertVedtak.sakId}. " +
+                    "Det betyr at vi ikke sender ut brev for vedtaket etter at en utbetaling går til oppdrag. " +
```

Comment:
```suggestion
                    "Det betyr at vi ikke sender ut brev for vedtaket. " +
```
Her er det ingen ting som skal gå til oppdrag (vi bør dobbeltsjekke at etterlatte-utbetaling håndterer at denne meldingen går ut uten at den prøver å iverksette / sende noe til oppdrag)

Reply:
Ok, sånn ja. Brevutsending og utbetaling trigges begge av kafka-meldingen. Me fix